src/ControlSystem/ControlErrors/Translation.hpp

```diff
@@ -72,6 +72,9 @@ namespace ControlErrors {
 struct Translation : tt::ConformsTo<protocols::ControlError> {
   static constexpr size_t expected_number_of_excisions = 2;
 
+  using object_centers =
+      domain::object_list<domain::ObjectLabel::A, domain::ObjectLabel::B>;
```

Comment:
this one only uses A I think?

Reply:
I think translation follows the COM of the total system, so uses A&B to get the COM of the binary

Reply:
Part of the translation control error calculation is the rotation and expansion control error calculations. So even though in this function itself it only uses A, rotation and expansion need both. Hence translation needs both.

Reply:
Right I missed this issue with the base tags